dev/README_FIRST.md

```diff
@@ -0,0 +1,226 @@
+# ⚠️ Warning
```

Comment:
BIDS documentation for the developers. Need to check if this should be moved to our wiki and/or ReadTheDocs before the PR is merged.

Reply:
I think this should be ho into the wiki page, we better keep RTD cleaner for users.